.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-max-line-length = 100
+max-line-length = 127
```

Comment:
Updated the max line length to `127`, as it is the max width of the Github preview.